internal/httphandler/ledger_request_handler.go

```diff
@@ -36,14 +36,18 @@ func NewLedgerRequestHandler(db bcdb.DB, logger *logger.SugarLogger) http.Handle
 	handler.router.HandleFunc(constants.GetBlockHeader, handler.blockQuery).Methods(http.MethodGet)
 	// HTTP GET "/ledger/path?start={startId}&end={endId}" gets shortest path between blocks
 	handler.router.HandleFunc(constants.GetPath, handler.pathQuery).Methods(http.MethodGet).Queries("start", "{startId:[0-9]+}", "end", "{endId:[0-9]+}")
-	// HTTP GET "/ledger/proof/{blockId}?idx={idx}" gets proof for tx with idx index inside block blockId
+	// HTTP GET "/ledger/proof/tx/{blockId}?idx={idx}" gets proof for tx with idx index inside block blockId
 	handler.router.HandleFunc(constants.GetTxProof, handler.txProof).Methods(http.MethodGet).Queries("idx", "{idx:[0-9]+}")
+	// HTTP GET "/ledger/proof/data/{blockId}/{dbname}/{key}?deleted={true|false}" gets proof for tx with idx index inside block blockId
+	handler.router.HandleFunc(constants.GetDataProof, handler.dataProof).Methods(http.MethodGet).Queries("deleted", "{deleted:true|false}")
```

Comment:
Is it possible to use this with a default `deleted=false` ?
deleted seems to be rare enough so that we allow a default for the frequent case.

Reply:
Have to thinks about it...

Reply:
Done